#variances of each state variable

float32 x
float32 y
float32 z

float32 Lx
float32 Ly
float32 Lz

float32 dx
float32 dy
float32 dz

float32 roll
float32 pitch
float32 yaw
float32 dyaw